tests/spec/directives/user-picker.spec.js

```diff
@@ -1009,6 +1009,43 @@ describe('luidUserPicker', function(){
 		});
 	});
 
+	/**************************
+	** CUSTOM HTTP SERVICE   **
+	***************************/
+	describe("with customHttpService", function(){
+		var chloe = { id:3,
+			firstName:"Chloé",
+			lastName:"Azibert Yekdah"
+		};
+		var customHttpService = {
+			get: function(query){
+				return $q.defer().promise;
+			}
+		};
```

Comment:
there are other calls to the get method done after the first get resolves, when there are homonyms. your tests does not cover that in that case `$http.get` isnt called